SolWatch/Game1.cs

```diff
@@ -58,19 +80,33 @@ protected override void Draw(GameTime gameTime)
         {
             GraphicsDevice.Clear(Color.Black);
 
-            // TODO: Add your drawing code here
             spriteBatch.Begin();
+
+            // Draw Sol
             spriteBatch.Draw(
                 texture: solTexture,
                 destinationRectangle: new Rectangle(
-                    location: solPosition,
-                    size: CelestialBodySymbolSize),
+                    location: screenCenter,
```

Comment:
I really like how readable specifying the parameters makes invoking this method in particular.